ksqldb-setup-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
I don't think this shouldn't be a StatefulSet because it will be restarted after the process shuts down.

Reply:
fair enough. CronJob then?

Reply:
We really only want to run it once to setup the tables & streams. So I think just a k8s `Job`.

Reply:
correct. i forget the exact name of this resource